src/components/orgs/Coupon/Container.tsx

```diff
@@ -155,7 +162,12 @@ const CouponContainer: React.FC<CouponContainerProps> = ({
           supplierName={supplierName}
         />
         <ButtonPrimary
-          onClick={() => handleActiveCoupon()}
+          onClick={() =>
+            handleActiveCoupon(
```

Comment:
What u pass a CONSTANT from params in "handleActiveCoupon"? 
Use this constant inside function is better, because isn't a user data.

Reply:
because we have the problem that we see in source debug. the Constant inside function was getting value undefined, só i put out of function and resolved.

Reply:
![image](https://github.com/Agility-Creative-Solutions/ez2live-app/assets/104227174/4a2b77d5-8716-4388-822c-e83248fbc6bd)